src/ext/Fintraffic/assets/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+  "short_name": "RAE",
+  "name": "Fintraffic Reitti- ja aikataulueditori",
+  "icons": [
+    {
+      "src": "favicon.ico",
+      "sizes": "32x32"
+    }
+  ],
+  "start_url": "./index.html",
+  "display": "standalone",
+  "theme_color": "#000000",
+  "background_color": "#ffffff"
+}
```

Comment:
This felt questionable, we could also override this at a build time. But maybe it's easier this way, just a link to plug in under <Helmet />